### DISCUSSION TITLE
Aggregation of multiple runs

### DISCUSSION THREAD

**Discussion by adrdufour** [aertslab/scenicplus#211] [Q&A]
Hi,

I want to perform multiple SCENIC+ analyses and aggregate the results of the different iterations similar to the multiple scenic runs in the VSN pipeline.

I have a few differences from different runs with the same datasets.

Thank you for your help,
Adrien Dufour.

**Comment by SeppeDeWinter** [comment]
Hi Adrien

Currently we don't have code to do this automatically. Is there any particular step where you are stuck with this analysis?
I would also be curious to know how much the variability is per run?

Best,

Seppe

**Comment by adrdufour** [comment]
Hi Seppe,

It's not a particular step; I want to recover some TF I lose when I re-run or add new datasets.
I get 59 regulons in the first round and 55 in the second runs with 20% of TF changes.
I have 33000 cells with nine libraries.

Best regards,
Adrien Dufour.
